### ISSUE TITLE
Can not pass hashes to set_info_plist_value

### ISSUE THREAD

**Issue by hammertoe** [fastlane/fastlane#4103]
Re-opening a clone of ticket that was just closed #913 

A PR was submitted but never accepted. Fix is a one-liner (but I think is_string is now deprecated, so not sure how best to handle this as you need to pass in a type and I want it to accept any type).

```
--- /Library/Ruby/Gems/2.0.0/gems/fastlane-1.67.0/lib/fastlane/actions/set_info_plist_value.rb  2016-03-23 14:44:18.000000000 +0000
+++ /Library/Ruby/Gems/2.0.0/gems/fastlane-1.68.0/lib/fastlane/actions/set_info_plist_value.rb  2016-03-30 16:23:31.000000000 +0100
@@ -34,6 +34,7 @@
           FastlaneCore::ConfigItem.new(key: :value,
                                        env_name: "FL_SET_INFO_PLIST_PARAM_VALUE",
                                        description: "Value to setup",
+                      is_string: false,
                                        optional: false),
           FastlaneCore::ConfigItem.new(key: :path,
                                        env_name: "FL_SET_INFO_PLIST_PATH",
```

**Comment by vpolouchkine** [comment]
Thanks for confirming @hammertoe. We'll look into this.

**Comment by mfurtak** [comment]
Hi @hammertoe - thanks for the report, and apologies about your previous PR not getting attention.

I've put your patch up as #4106 :+1:

**Comment by mfurtak** [comment]
This will go out in the next `fastlane` release! :rocket:

**Comment by hammertoe** [comment]
Awesome thanks! :)

-Matt

> On 7 Apr 2016, at 18:47, Michael Furtak <notifications@github.com> wrote:
> 
> Closed #4103 <https://github.com/fastlane/fastlane/issues/4103>.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/fastlane/fastlane/issues/4103#event-618877022>


— 
Matt Hamilton
Quernus
matt@quernus.co.uk
+44 117 325 3025
49b Easton Business Centre
Felix Road, Easton
Bristol, BS5 0HE

Quernus Ltd is a company registered in England and Wales. Registered number: 09076246
